Assets/Scripts/UI/Canvas/StateMachine/UIStateCaller.cs

```diff
@@ -0,0 +1,21 @@
+using UnityEngine;
+
+namespace UI.Canvas.StateMachine
+{
+    public sealed class UIStateCaller : MonoBehaviour
+    {
+        [SerializeField] private UIStateManager stateManager;
+
+        private UIStateEvent onStateChange;
```

Comment:
```suggestion
        private UIStateEvent _onStateChange;
```
[code convention](https://github.com/Team-Swamp/CaptureGraafsmeer/wiki/Code-convention#variables)